.travis.yml

```diff
@@ -1,10 +1,8 @@
 language: python
 python:
-  - '3.6'
-
+- '3.6'
```

Comment:
Is it a voluntary change to remove all indents? I kind of want to keep them for readability reason.

Reply:
oh. hmm, I used an auto-formatter for yaml, and this was the output. I'll try to configure it to have indents

Reply:
should be fixed now.